services/QuillLMS/client/app/bundles/Shared/styles/input.scss

```diff
@@ -339,10 +336,9 @@
       border-style: none none solid none;
     }
     label {
-      color: #7f7f7f;
-      font-size: 16px;
+      color: $quill-grey-90;
+      @include display-xs;
```

Comment:
I think we might want to make a pattern of putting these mixins at the top of the style definition? What do you think?

Reply:
I think that makes sense!